cmd/serve/dlna/dlnaflags/dlnaflags.go

```diff
@@ -36,6 +39,7 @@ var DefaultOpt = Options{
 	FriendlyName:   "",
 	LogTrace:       false,
 	InterfaceNames: []string{},
+	AnnounceInterval: 10 * time.Second,
```

Comment:
Feel free to increase the default if you think it is wrong.

Reply:
I think, increasing the default value to 12 minutes, so according to the upstream SSDP [code](https://github.com/anacrolix/dms/blob/cf5ba6b1180099f2fe8dbec232ad10f072fb5d13/ssdp/ssdp.go#L174), the cache timeout is 1800 seconds, which may be more appropriate.

Reply:
Will this increase the time for clients to find the `rclone dlna` server?

Reply:
Thanks for explaining. This change sounds good.